code/modules/cargo/packs/sec_supply.dm

```diff
@@ -50,6 +50,52 @@
 	contains = list(/obj/item/circuitboard/computer/security)
 	crate_name = "camera console crate"
 
+/*
+		Pouches
+*/
+
+/datum/supply_pack/sec_supply/pouch
+	name = "General Pouch Crate"
+	desc = "Contains a small basic pouch for holding two small items of your choice."
+	cost = 250
```

Comment:
This should probably be closer to 100-150. It doesn't come with anything in it.

I'd likely do this:

150 for pouches that come empty, 250 for pouches that come with equipment.